lib/evm_database.rb

```diff
@@ -32,7 +32,11 @@ def self.seedable_model_class_names
   end
 
   def self.seed_primordial
-    seed(PRIMORDIAL_CLASSES)
+    if ENV['SKIP_PRIMORDIAL_SEED'] == 'true' && MiqDatabase.count > 0
```

Comment:
Sorry to nitpick, but if this is to behave like other boolean settings we have it should be `if ENV['SKIP_PRIMORDIAL_SEED'] && ...` (don't check `== 'true'`, any truthy value is valid, such as `1`)